db/seeds.rb

```diff
@@ -8,20 +8,25 @@
 Event.destroy_all
 Config.destroy_all
 
+# Create google SSO user
```

Comment:
Why is this commented out? Is this documentation? If so, can you move it to the docs?